api/application_submissions/index.js

```diff
@@ -0,0 +1,59 @@
+const express = require('express');
+
+const router = express.Router();
+
+const ApplicationSubmission = require('../../models/applicationSubmissionsModel');
+
+router.get('/:id', async (req, res) => {
+  try {
+    const { id } = req.params;
+    const applicationSubmission = await ApplicationSubmission.findById(id);
+    if (applicationSubmission) {
+      res.status(200).json({ applicationSubmission, error: null });
+    } else {
+      res.status(400).json({ message: 'Submission not found in the database' });
```

Comment:
Make this a 404 please (https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_Client_errors)